test/rpc/test_rpc_class.py

```diff
@@ -20,12 +20,27 @@ def test_get_class(class_hash):
     contract_class = resp["result"]
 
     assert contract_class["entry_points_by_type"] == {
-        "CONSTRUCTOR": [],
-        "EXTERNAL": [
-            {"offset": "0x3a", "selector": "0x0362398bec32bc0ebb411203221a35a0301193a96f317ebe5e40be9f60d15320"},
-            {"offset": "0x5b", "selector": "0x039e11d48192e4333233c7eb19d10ad67c362bb28580c604d67884c85da39695"}
-        ],
-        "L1_HANDLER": []
+        "CONSTRUCTOR": [
```

Comment:
Can this be extracted somehow instead of having it copied in 2 places?
https://github.com/Shard-Labs/starknet-devnet/pull/227/files#diff-ffa0bbaa73d5171327e5293e41e360ddfe123c2c8a0bbf47dd27c48b948ec569R81

Reply:
oh yeah i think it'd neater that way

on it!